frames/src/main/java/com/checkout/frames/style/theme/PaymentFormTheme.kt

```diff
@@ -10,15 +10,26 @@ public data class PaymentFormTheme(
      */
     val paymentFormComponents: List<PaymentFormComponent> = DefaultPaymentFormTheme.provideComponents(),
     /**
-     * Custom CornerRadius provided by the merchants
+     * Custom Shape provided by the merchants
      */
-    val paymentFormCornerRadius: PaymentFormCornerRadius = PaymentFormCornerRadius(),
+    val paymentFormShape: PaymentFormShape,
     /**
-     * Custom Shape provided by the merchants
+     * Custom CornerRadius provided by the merchants
      */
-    val paymentFormShape: PaymentFormShape = PaymentFormShape(),
+    val paymentFormCornerRadius: PaymentFormCornerRadius,
     /**
      * Custom Font provided by the merchants
      */
-    val paymentFormFont: PaymentFormFont = PaymentFormFont()
-)
+    val paymentFormFont: PaymentFormFont,
+) {
+    public constructor(
+        paymentFormThemeColors: PaymentFormThemeColors,
+        paymentFormComponents: List<PaymentFormComponent>,
+    ) : this(
+        paymentFormThemeColors,
+        paymentFormComponents,
+        PaymentFormShape(),
+        PaymentFormCornerRadius(),
+        PaymentFormFont()
+    )
```

Comment:
This constructor doesn't extend functionality but limits it, so in the end, only this constructor is available for use.
To improve it just remove that secondary constructor and set default values.